From 351bfba20540881449134a93a0fb45a349238cfb Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Wed, 2 Apr 2014 11:56:50 +0200
Subject: [PATCH] skip selenium tests, when WITH_SELENIUM is not set

It was possible, a selenium test case was called,
even if it wasn't specified.
Closes-bug: #1241027

Change-Id: Id2547191d534efe4c26dd7439dff59fa773255eb
---
 horizon/test/tests/selenium.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/horizon/test/tests/selenium.py b/horizon/test/tests/selenium.py
index efe0439..4ef58e4 100644
--- a/horizon/test/tests/selenium.py
+++ b/horizon/test/tests/selenium.py
@@ -14,9 +14,14 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from horizon.test import helpers as test
+from os import environ
+
+from django.utils import unittest
 
+from horizon.test import helpers as test
 
+@unittest.skipIf(environ.get('WITH_SELENIUM', False),
+        "test called without WITH_SELENIUM")
 class BrowserTests(test.SeleniumTestCase):
     def test_qunit(self):
         self.selenium.get("%s%s" % (self.live_server_url, "/qunit/"))
-- 
1.9.0

